index.js

```diff
@@ -9,6 +9,9 @@ module.exports = {
   parserOptions: {
     ecmaVersion: 2017,
     sourceType: `module`,
+    ecmaFeatures: {
+      experimentalObjectRestSpread: true,
```

Comment:
This shouldn't be needed anymore; `ecmaVersion: 2018` ought to do it

https://eslint.org/docs/user-guide/configuring#deprecated: "ecmaFeatures.experimentalObjectRestSpread - enable support for the experimental object rest/spread properties. This syntax has been supported in ecmaVersion: 2018. This option will be removed in the future."